### PR TITLE
Updated documentation for the road domain plus fixing Pset applicability

### PR DESCRIPTION
**Pull request by larswik**
Hopefully standard references are made in the correct way. I also noticed that Pset applicability were either missing or wrong (such as referencing "IfcSurf"?). I fixed all these as well.

### CODE REVIEW

File: IFC4x3/PropertyConstants/b/BOTH_3sytttEkrC8OE_plAGIwDS.xml [hunk 1]
```diff
@@ -1,3 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BOTH_3sytttEkrC8OE_plAGIwDS" Name="BOTH" UniqueId="f6f37df7-3aed-4c21-83be-cef2904ba35c" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BOTH_3sytttEkrC8OE_plAGIwDS" Name="BOTH" UniqueId="f6f37df7-3aed-4c21-83be-cef2904ba35c">
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/b/BOTH_3sytttEkrC8OE_plAGIwDS.xml:2]: We are going to have merge conflicts as @jmirtsch  also pulled this in.

**Review by SergejMuhic — COMMENTED** [review]
1. need to coordinate with @jmirtsch which PR to merge as he also pulled in the changes for constants
2. '>NOTE&nbsp;' should be used as 'NOTE&amp;nbsp;' produces an ampersand and nbsp; i.e. '&nbsp;'. I guess this was mistakenly converted a couple of times in IfcDoc. Something we have to address.

File: IFC4x3/PropertyConstants/l/LINEAR_1m2i15jzb7DeT8TSDmT03v.xml [hunk 2]
```diff
@@ -1,3 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LINEAR_1m2i15jzb7DeT8TSDmT03v" Name="LINEAR" UniqueId="700ac045-b7d9-4736-8748-75c3707400f9" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LINEAR_1m2i15jzb7DeT8TSDmT03v" Name="LINEAR" UniqueId="700ac045-b7d9-4736-8748-75c3707400f9">
+	<Localization>
+		<DocLocalization Locale="en">
+			<Documentation>Specifies that the transition of the superelevation between the previous superelevation event and this event shall be linearly interpolated.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/PropertyConstants/l/LINEAR_1m2i15jzb7DeT8TSDmT03v.xml:5]: Merge Conflict - suggestion in other pull request was to combine both LINEAR constants

**Review by AlexBrad1eyCT — APPROVED** [review]

**Comment by SergejMuhic** [comment]
> Need to decide on the merge conflict strategy, I do not see any changes needed to this pull request other than the conflicts can i suggest merging this pull request then managing the conflicts in @jmirtsch 's pull request #23 ??

In principle, I would agree as @jmirtsch PR also adds some other stuff. I would ask @jmirtsch though to give his insight. Maybe he can just cherry-pick out the stuff that was not added manually in other PRs.

**Comment by AlexBrad1eyCT** [comment]
@jmirtsch, we want to merge this request but just want to check with you that you can handle the merge conflict in #23 , I will help in identifying them but obvously only you have write access to your fork!

**Pull request merged**